docs/src/main/asciidoc/health-guide.adoc

```diff
@@ -56,141 +58,171 @@ which is an implementation of the MicroProfile Health specification used in Quar
     public HealthCheckResponse call() {
         return HealthCheckResponse.named("Simple health check").up().build();
     }
 }
 ----
 
-As you can see health check procedures are defined as implementations of the `HealthCheck` 
-interface which are defined as CDI beans with the `@Health` qualifier. `HealthCheck` is 
-a functional interface whose single method `call` returns a `HealthCheckResponse` object 
-which can be easily constructed by the fluent builder API shown in the example.
+As you can see health check procedures are defined as implementations of the 
+`HealthCheck` interface which are defined as CDI beans with the one of the 
+following CDI qualifiers: 
+
+- `@Liveness` - the liveness check accessible at `/health/live`
+- `@Readiness` - the readiness check accessible at `/health/ready`
+- `@Health` - *deprecated* unspecified health procedure accessible at `/health`
+
+WARNING: The `@Health` qualifier is deprecated in version 2.0 of MicroProfile 
+Health specification and it can be removed in any later release.
+
+NOTE: The `/health` endpoint accumulates all `@Liveness`, `@Readiness` and deprecated 
+`@Health` procedures and it is possible it will be removed in a later release of the 
+specification.  
+
+`HealthCheck` is a functional interface whose single method `call` returns a 
+`HealthCheckResponse` object which can be easily constructed by the fluent builder 
+API shown in the example.
 
 As we have started our Quarkus application in dev mode simply repeat the request 
-to `http://localhost:8080/health` by refreshing your browser window or by using `curl http://localhost:8080/health`. 
-The new health check procedure is now present in the `checks` array.
+to `http://localhost:8080/health/live` by refreshing your browser window or by 
+using `curl http://localhost:8080/health/live`. Because we defined our health check 
+to be a liveness procedure (with `@Liveness` qualifier) the new health check procedure 
+is now present in the `checks` array.
 
 Congratulations! You've created your first Quarkus health check procedure. Let's 
 continue by exploring what else can be done with the MicroProfile Health specification.
 
-== Adding user specific data to the health check response
+== Adding a readiness health check procedure
 
-In previous section we saw how to create a simple health check with only the minimal 
-attributes, namely, the health check name and its state (UP or DOWN). However, the 
-MicroProfile specification also provides a way for the applications to supply 
-arbitrary data in the form of key value pairs sent to the consuming end. This can be done by using the 
-`withData(key, value)` method of the health check response builder API.
+In the previous section, we created a simple liveness health check procedure which states 
+whether our application is running or not. In this section, we will create a readiness 
+health check which will be able to state whether our application is able to process 
+requests.
```

Comment:
I must admit I would talk about readiness first as it seems more logical.

But maybe there's a good reason for this order?